src/NodeServices/IAnalyticsClient.cs

```diff
@@ -433,7 +433,27 @@ public enum Actions
         /// <summary>
         /// ExportSettings event, e.g. tracks the ExportSettings event
         /// </summary>
-        ExportSettings
+        ExportSettings,
+
+        /// <summary>
+        /// Ajusting the confidence level threshold in the Preferences panel
+        /// </summary>
+        AdjustConfidenceLevel,
```

Comment:
We can probably re-use `Set` instead of creating a new one, what do you think?

Reply:
This Action was removed and instead is using the Set Action, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13581/commits/0655f6d242fa449dc828a75d8634567d3b5737f4